metricbeat/docs/index.asciidoc

```diff
@@ -1,5 +1,6 @@
 = Metricbeat Reference
 :libbeat: http://www.elastic.co/guide/en/beats/libbeat/master
+:filebeat: http://www.elastic.co/guide/en/beats/filebeat/master
```

Comment:
No, this variable is used to resolve links that point to the filebeat docs. :-) In the regexp doc, we have a link that points to the filebeat docs for some examples of regexp patterns (the patterns are for multiline, but they might be useful to people in general who are looking for example regexp patterns).